service/src/main/java/uk/gov/hmcts/reform/fpl/service/UserDetailsService.java

```diff
@@ -23,4 +24,8 @@ public UserDetailsService(IdamApi idamApi) {
     public String getUserName(String authorization) {
         return idamApi.retrieveUserDetails(authorization).getFullName();
     }
+
+    public UserDetails getUserDetails(final String authorization) {
```

Comment:
seems silly to have a getUserName method and a getUserDetials method when they are doing the same thing... how much effort is it to refactor the use of the method above to return UserDetails?

Reply:
you are right. Making the change here will affect 9-classes. Not sure if we want to make as part of this PR?

Reply:
you want to create a tech debt ticket in jira for this?

Reply:
Done [here](https://tools.hmcts.net/jira/browse/FPLA-850)